hnn_core/dipole.py

```diff
@@ -145,6 +145,101 @@ def simulate_dipole(net, n_trials=1, n_jobs=1):
     return dpl
 
 
+def average_dipoles(dpls):
```

Comment:
I am wondering if we should have an `__add__` method in dipoles. That can be used for this and also for the rmse.

Reply:
Maybe later

Reply:
@jasmainak can you describe whay would happen in `__add__`? would RMSE get calculated automatically?

Reply:
ummm ... I am not sure any more what I meant, I think it's fine to resolve this comment.